python/ray/train/pytorch_lightning/pytorchlightning_trainer.py

```diff
@@ -0,0 +1,64 @@
+from typing import Type, Option​​al, Dict
+
+from ray.util import PublicAPI
+from ray.train.torch import TorchTrainer, TorchConfig
+from ray.air.config import ScalingConfig, DatasetConfig, RunConfig
+from ray.train.trainer import GenDataset
+from ray.data.preprocessor import Preprocessor
+from ray.air.checkpoint import Checkpoint
+
+
+@PublicAPI(stability="alpha")
+class PytorchLightningTrainer(TorchTrainer):
```

Comment:
How about let's add the docstrings now, and add an example script?

Also rename everything to Lightning instead of PytorchLightning?